src/main/java/org/fundacion/jala/converter/core/facade/CompilerFacade.java

```diff
@@ -26,13 +31,43 @@
     public CompilerFacade() {
     }
 
+    /**
+     * Compiles a Java file.
+     *
+     * @param code is string with code in java.
+     * @return a string of the result on runtime console.
+     */
+    public static String facadeJavaCompile(final String code) {
+        if (!code.isBlank() || !code.equals(null)) {
+            javaCompiler = new JavaCompiler();
+            String filePath = Transform.toFile(code, "Main", "java");
+            return javaCompiler.javaCompiler(new JavaParameter(JavaVersion.JAVA_11, filePath));
+        }
+        return "";
+    }
+
+    /**
+     * Compiles a Python file.
+     *
+     * @param code is string with code in Python.
+     * @return a string of the result on runtime console.
+     */
```

Comment:
in the ereturn javadoca a string must be uppercase to String

Reply:
you are rigth